backend/src/api_parsers/scopus_parser.py

```diff
@@ -46,6 +46,7 @@ def _parse_entry_dict(self, entry: dict) -> None:
             "title": entry["dc:title"],
             "abstract": entry.get("dc:description"),
             "citations": entry.get("citedby-count"),
+            "venue": '',
```

Comment:
Can we make the venue optional in model and not pass it at all?

Reply:
will do in a separate task